corehq/apps/userreports/views.py

```diff
@@ -707,7 +707,8 @@ def export_data_source(request, domain, config_id):
     # First row is taken up by headers
     if params.format == Format.XLS and q.count() >= 65535:
         keyword_params = dict(**request.GET)
-        keyword_params.update(format=Format.CSV)
+        # use default format
+        del keyword_params['format']
```

Comment:
I think it would always exist because the format was set to XLS - but sounds safer to check, in case the rest of the code changes.  fixed.